### DISCUSSION TITLE
can't start default

### DISCUSSION THREAD

**Discussion by tensiontocoding** [lima-vm/lima#640] [Q&A] [answered]
sorry i'm not good at english, but i need help!

i installed lima now, following the guide, to install docker and oracle.

but every time, when i try to start default or other instance

error occur at  [hostagent]Waiting for the essential requirement 3 of 5: "sshfs binary to be installed"

so why is it occur??

how can i fix it??

? Creating an instance "default" Proceed with the default configuration
INFO[0002] Attempting to download the image from "https://cloud-images.ubuntu.com/impish/current/impish-server-cloudimg-arm64.img"  digest=
537.00 MiB / 537.00 MiB [-----------------------------------] 100.00% 6.85 MiB/s
INFO[0083] Downloaded image from "https://cloud-images.ubuntu.com/impish/current/impish-server-cloudimg-arm64.img"
INFO[0083] [hostagent] Replacing "http_proxy" value "http://127.0.0.1:8000" with "http://192.168.5.2:8000"
INFO[0083] [hostagent] Replacing "https_proxy" value "http://127.0.0.1:8000" with "http://192.168.5.2:8000"
INFO[0083] [hostagent] Starting QEMU (hint: to watch the boot progress, see "/Users/yt/.lima/default/serial.log")
INFO[0083] SSH Local Port: 60022
INFO[0083] [hostagent] Waiting for the essential requirement 1 of 5: "ssh"
INFO[0141] [hostagent] The essential requirement 1 of 5 is satisfied
INFO[0141] [hostagent] Waiting for the essential requirement 2 of 5: "user session is ready for ssh"
INFO[0141] [hostagent] The essential requirement 2 of 5 is satisfied
INFO[0141] [hostagent] Waiting for the essential requirement 3 of 5: "sshfs binary to be installed"
INFO[0181] [hostagent] Waiting for the essential requirement 3 of 5: "sshfs binary to be installed"
INFO[0222] [hostagent] Waiting for the essential requirement 3 of 5: "sshfs binary to be installed"
INFO[0262] [hostagent] Waiting for the essential requirement 3 of 5: "sshfs binary to be installed"
INFO[0302] [hostagent] Waiting for the essential requirement 3 of 5: "sshfs binary to be installed"
INFO[0342] [hostagent] Waiting for the essential requirement 3 of 5: "sshfs binary to be installed"
INFO[0382] [hostagent] Waiting for the essential requirement 3 of 5: "sshfs binary to be installed"
INFO[0422] [hostagent] Waiting for the essential requirement 3 of 5: "sshfs binary to be installed"
INFO[0462] [hostagent] Waiting for the essential requirement 3 of 5: "sshfs binary to be installed"
INFO[0502] [hostagent] Waiting for the essential requirement 3 of 5: "sshfs binary to be installed"
INFO[0542] [hostagent] Waiting for the essential requirement 3 of 5: "sshfs binary to be installed"
INFO[0583] [hostagent] Waiting for the essential requirement 3 of 5: "sshfs binary to be installed"
INFO[0623] [hostagent] Waiting for the essential requirement 3 of 5: "sshfs binary to be installed"
INFO[0663] [hostagent] Waiting for the essential requirement 3 of 5: "sshfs binary to be installed"
FATA[0683] did not receive an event with the "running" status <------------------------------------------ always occur

please... help me

**Comment by DennisRasey** [comment, marked as answer]
Any luck getting this working?

I'm completely guessing here, but:

Lima will try to download the sshfs package for Impish.
Could this be due to your Internet connection or VPN not allowing the download (my corporate VPN messes up some type of downloads).

You might be able to access the "failing" vm in another terminal (limactl shell default) to troubleshoot.

**Reply by tensiontocoding** [reply]
thanks
i think unicorn vpn interrupt the install
so remove vpn and problem is solved!

**Comment by afbjorklund** [comment]
You want to look at the serial console log (on the host) and eventually at the cloud-init log (available inside the VM)

`INFO[0083] [hostagent] Starting QEMU (hint: to watch the boot progress, see "/Users/yt/.lima/default/serial.log")`
